src/domains/sign/cosmos.tf

```diff
@@ -1,49 +1,19 @@
 module "cosmosdb_account" {
-  source                           = "git::https://github.com/pagopa/azurerm.git//cosmosdb_account?ref=v2.13.1"
-  name                             = "${local.project}-cosmos"
+  source                           = "git::https://github.com/pagopa/azurerm.git//cosmosdb_account?ref=v2.15.1"
+  name                             = format("%s-cosmos", local.project)
   location                         = azurerm_resource_group.data_rg.location
   resource_group_name              = azurerm_resource_group.data_rg.name
   kind                             = "GlobalDocumentDB"
   main_geo_location_location       = azurerm_resource_group.data_rg.location
-  main_geo_location_zone_redundant = false
+  main_geo_location_zone_redundant = var.cosmos.zone_redundant
```

Comment:
add a secondary region

Reply:
Looks like there are not enough azure resources in North Europe

```
Sorry, we are currently experiencing high demand in North Europe region for the zonal redundant (Availability Zones) accounts, and cannot fulfill your request at this time Wed, 14 Dec 2022 17:56:15 GMT. To request region access for your subscription, please follow this link https://aka.ms/cosmosdbquota for more details on how to create a region access request.
```

Reply:
I propose to stick with one region for now